routes/search.js

```diff
@@ -23,16 +26,20 @@ router.get('/', common.isAuthenticated, (req, res, next) => {
     })
 })
 
+router.get('/faq', common.isAuthenticated, (req,res,next) => {
```

Comment:
I removed this from search and put it in index in my last PR. I don't think this should be in the search file as it ends with a weird route of "/search/faq" where as in the index file it will display as "/faq". Plus this page is not specific to search, it's entire application relevant.